try {
  console.log("Hello, world!");
} catch (err: any) {
  throw err;
}
